_data/education.yml

```diff
@@ -1,6 +1,6 @@
 # Degree
-- degree: Full Stack Webdeveloper Nanodegree
+- degree: Full Stack Webdeveloper Nanodegree  
   uni: Udacity
   year: June, 2014 &mdash; January, 2015
-  # awards: 
-  # summary: 
+  # awards:
+  # summary:
```

Comment:
Should be commented out and let the person responsible know if they would like to change it.